brew_view/static/src/js/services/system_service.js

```diff
@@ -32,12 +32,21 @@ export default function systemService($http) {
     return $http.delete('api/v1/systems/' + system.id);
   };
 
-  SystemService.getSystems = function() {
-    return $http.get('api/v1/systems');
+  SystemService.getSystems = function(dereference_nested, includeFields,
+      excludeFields) {
+    return $http.get('api/v1/systems', {
+      params: {
+        dereference_nested: dereference_nested,
+        include_fields: includeFields,
+        exclude_fields: excludeFields,
+      }
+    });
   };
 
   SystemService.getSystem = function(id, includeCommands) {
-    return $http.get('api/v1/systems/' + id, {params: {include_commands: includeCommands}});
+    return $http.get('api/v1/systems/' + id, {
+      params: {include_commands: includeCommands}
```

Comment:
If this is deprecated, we should not be using it. It feels like we could maybe translate this (so the JS API doesn't have to change) but we convert this to the "correct" usage.

Reply:
Hmm. I can add the same kind of filtering to the individual system GET if you think that's necessary. I agree, it would be kind of weird to have that parameter be deprecated on the list API and not on the single system API.

Reply:
Oh interesting. I didn't realize this was on the System API

I agree its weird, but I don't think I care enough to fix it right now.

We need a more full-fledged solution for handling exclusion/inclusions. So we can tackle it there.